source/slang/slang-ast-support-types.h

```diff
@@ -1181,6 +1181,7 @@ namespace Slang
         /// "under-construction" and not being checked, then it's safe to
         /// consider all names we've inserted so far. This is used when
         /// checking to see if a keyword is shadowed.
+        IgnoreInheritance = 1 << 4, //< Lookup only non inheritance children of a struct (including `extension`)
```

Comment:
You need three slashes for the Doxygen comment.
See the document of [Doxygen](https://www.doxygen.nl/manual/docblocks.html) "Putting documentation after members" section.